tcms_tenants/templates/tcms_tenants/new.html

```diff
@@ -62,6 +62,16 @@
             {% block extra_contents %}
                 {{ form.paid_until }}
             {% endblock %}
+
+            {% if object %}
+                <div class="form-group">
+                    <label class="col-md-1 col-lg-1">{% trans "Logo" %}</label>
+                    <div class="col-sm-11 col-md-8 col-lg-5">
+                        {% include 'include/attachments.html' %}
+                    </div>
+                </div>
+           {% endif %}
```

Comment:
This entire block needs to be after the `</form>` HTML tag b/c the attachments form has it's own `<form>` and mixing them up messes things up.

However when you move this block things start to look a bit ugly.